pipelines-it-remote.yml

```diff
@@ -0,0 +1,64 @@
+jobs:
+- job: 'Ubuntu_16_04_remote'
+  pool: 'NNI CI KUBE CLI'
+
+  steps:
+  - script: python3 -m pip install --upgrade pip setuptools
+    displayName: 'Install python tools'
+  - script: |
+      source install.sh
+    displayName: 'Install nni toolkit via source code'
+  - task: CopyFilesOverSSH@0
+    inputs:
+      sshEndpoint: remote_nni-ci-gpu-01
+      sourceFolder: src/sdk/pynni
+      targetFolder: /tmp/nnitest/$(Build.BuildId)/pynni
+      overwrite: true
+    displayName: 'Copy sdk files to remote machine'
+  - task: CopyFilesOverSSH@0
+    inputs:
+      sshEndpoint: remote_nni-ci-gpu-01
+      sourceFolder: tools
+      targetFolder: /tmp/nnitest/$(Build.BuildId)/tools
+      overwrite: true
+    displayName: 'Copy tool files to remote machine'
+  - task: CopyFilesOverSSH@0
+    inputs:
+      sshEndpoint: remote_nni-ci-gpu-01
+      sourceFolder: test
+      targetFolder: /tmp/nnitest/$(Build.BuildId)/test
+      overwrite: true
+    displayName: 'Copy test files to remote machine'
+  - task: SSH@0
+    inputs:
+      sshEndpoint: remote_nni-ci-gpu-01
+      runOptions: commands
+      commands: python3 /tmp/nnitest/$(Build.BuildId)/test/remote_docker.py --mode start --name $(Build.BuildId) --image nni/nni
+    displayName: 'Start docker'
+  - task: DownloadSecureFile@1
+    inputs:
+      secureFile: remote_ci_private_key
+  - script: |
+      mkdir ~/.ssh
+      ssh-keyscan $(remote_host) >> ~/.ssh/known_hosts
```

Comment:
Will this add one line into know_hosts for each build?

Reply:
removed